frontend/src/app/components/modal/instances/change-status-modal.tsx

```diff
@@ -114,6 +115,19 @@ export const ChangeStatusModal: FC<ChangeStatusModalProps> = ({ close, submit, c
             </Col>
           </Row>
         )}
+        {!is_officer_assigned && (
```

Comment:
Can we add check that the selectedStatus is closed? As it is, the warning text comes up even if closed isn't selected